src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,37 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Answers {
+    private final List<Answer> answers = new ArrayList<>();
+
+    public void add(Answer answer) {
+        answers.add(answer);
+    }
+
+    public List<Answer> getAnswers() {
+        return answers;
+    }
+
+    public void validateOwnership(NsUser loginUser) throws CannotDeleteException {
+        for (Answer answer : answers) {
+            if (!answer.isOwner(loginUser)) {
+                throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+            }
+        }
```

Comment:
```suggestion
        var hasNonOwner = answers.stream()
                .anyMatch(answer -> !answer.isOwner(loginUser));

        if (hasNonOwner) {
            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
        }
```

이렇게 stream을 활용해도 좋겠네요~

Reply:
Java 에서도 var를 사용할 수 있었군요...! depth 조건을 위배했네요. 수정하였습니다. 4940a19c8d3e22fbe606f3bb4efc0ddfdc409847